examples/step-47/doc/results.dox

```diff
@@ -211,3 +211,81 @@ make sense:
   addition should not be overly difficult using, for example, the
   FEInterfaceValues class combined with MeshWorker::mesh_loop() in the
   same spirit as we used for the assembly of the linear system.
+
+
+  <h3> Derivation for the simply supported plates </h3>
+
+  Similar to the “clamped” boundary condition addressed in the implementation,
+  we will derive the $C^0$ IP finite element scheme for the simply supported plates:
+  @f{align*}{
+    \Delta^2 u(\mathbf x) &= f(\mathbf x)
+    \qquad \qquad &&\forall \mathbf x \in \Omega,
+    u(\mathbf x) &= g(\mathbf x) \qquad \qquad
+    &&\forall \mathbf x \in \partial\Omega, \\
+    \Delta u(\mathbf x) &= h(\mathbf x) \qquad \qquad
+    &&\forall \mathbf x \in \partial\Omega.
+  @f}
+  We multiply the biharmonic equation by the test function $v_h$ and integrate over $\Omega$ and get:
+  @f{align*}{
+    \int_K v_h (\Delta^2 u_h)
+     &= \int_K (D^2 v_h) : (D^2 u_h)
+       + \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}}
+       -\int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}).
+  @f}
+
+  Summing up over all cells $K \in  \mathbb{T}$,
+  @f{align*}{
+  \sum_{K \in \mathbb{T}} \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}} = 0,
+  @f}
+  and by the definition of jump over cells,
+  @f{align*}{
+  -\sum_{K \in \mathbb{T}} \int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}) = -\sum_{e \in \mathbb{F}} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}}).
+  @f}
+  We separate interior faces and boundary faces of the domain,
+  @f{align*}{
+  -\sum_{K \in \mathbb{T}} \int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}) = -\sum_{e \in \mathbb{F}^i} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}})
+  - \sum_{e \in \partial \Omega} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} h,
+  @f}
+  Where $\mathbb{F}^i$ is the set of interior faces.
```

Comment:
```suggestion
  where $\mathbb{F}^i$ is the set of interior faces.
```

Reply:
Is this the symbol we had previously used?

Reply:
I suppose in the introduction section, we has boundary interfaces $\mathbb{F}^b$ but no interior interfaces. So I set interior faces to be $\mathbb{F}^i$.

Reply:
That's fine, but it should be explained somewhere. (If $F^b$ is not explained yet, would you mind adding that too where it is appropriate?)

Reply:
Shall we explain both of these two in the introduction section where the first place we use notation $\mathbb{F}$?

Reply:
Yes, I think that would be best. (If you don't use a symbol right away but want to introduce it there, you can always say "We will also denote by X the set of ... for use further down below.")

Reply:
Ok, I see. Thanks!